package.json

```diff
@@ -76,7 +80,7 @@
     "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/ || exit 0",
     "test": "node node_modules/.bin/jest",
     "test:watch": "node node_modules/.bin/jest --watchAll",
-    "storybook": "start-storybook --dont-track -p 6006",
```

Comment:
sadly that option has been removed, I was unable to find if it actually tracks or not, based on my browser connections it seems like it does not.